src/Components/Modal/Series.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "styled-components";
+
+const Series = props => {
+  console.log(props.data);
+  const { episode, detail_description, running_time, detail_thumb_nail } =
+    props.data;
+
+  return (
+    <ContentSeries>
+      <SeriesInfo>
+        <SeriesImg src={detail_thumb_nail} alt="img" />
+        <SeriesDetail>
+          <H3>{episode}</H3>
+          <SeriesDes>
+            <H4>{detail_description}</H4>
+          </SeriesDes>
+        </SeriesDetail>
+        <H4>{running_time}min</H4>
+      </SeriesInfo>
+    </ContentSeries>
+  );
+};
+
+export default Series;
+
+const text = styled.p`
```

Comment:
```suggestion
const Text = styled.p`
```
대문자로 바꿔주시는게 맞는 것 같아요

Reply:
✅ 수정완료했습니다!